ens/tests/.bin/ens-registry.wat

```diff
@@ -0,0 +1,12068 @@
+(module
+  (type (;0;) (func (param i32) (result i32)))
+  (type (;1;) (func (param i32 i32) (result i32)))
+  (type (;2;) (func))
+  (type (;3;) (func (param i32 i32)))
+  (type (;4;) (func (param i32 i32 i32)))
+  (type (;5;) (func (param i32 i32 i32) (result i32)))
+  (type (;6;) (func (result i32)))
+  (type (;7;) (func (param i32 i32 i32 i32) (result i32)))
+  (type (;8;) (func (param i32 i32 i32 i32)))
+  (type (;9;) (func (param i32)))
+  (type (;10;) (func (param i32 i32 i32 i32 i32 i32 i32)))
```

Comment:
delete the .bin folder & add to gitignore.

Reply:
done